File: easy/testing.hs
import System.Environment (getArgs)
import Data.List.Split (splitOn)

countBugs             :: [String] -> Int
countBugs [[], []]     = 0
countBugs [x:xs, y:ys] | x /= y    = succ (countBugs [xs, ys])
                       | otherwise = countBugs [xs, ys]

prioFix  :: Int -> String
prioFix 0 = "Done"
prioFix x | x <= 2    = "Low"
          | x <= 4    = "Medium"
          | x <= 6    = "High"
          | otherwise = "Critical"

main :: IO ()
main = do
    [inpFile] <- getArgs
    input <- readFile inpFile
    putStr . unlines . map (prioFix . countBugs . splitOn " | ") $ lines input

File: easy/testing.go
package main

import (
	"bufio"
	"fmt"
	"log"
	"os"
	"strings"
)

func test(s, t string) string {
	var r int
	for i := range s {
		if s[i] != t[i] {
			r++
			if r > 6 {
				break
			}
		}
	}
	switch {
	case r == 0:
		return "Done"
	case r <= 2:
		return "Low"
	case r <= 4:
		return "Medium"
	case r <= 6:
		return "High"
	default:
		return "Critical"
	}
}

func main() {
	data, err := os.Open(os.Args[1])
	if err != nil {
		log.Fatal(err)
	}
	defer data.Close()
	scanner := bufio.NewScanner(data)
	for scanner.Scan() {
		s := strings.Split(scanner.Text(), " | ")
		fmt.Println(test(s[0], s[1]))
	}
}
